Imperatus.rte/Devices/Weapons/Marauder/Marauder.ini

```diff
@@ -167,16 +167,20 @@ AddDevice = HDFirearm
 			FilePath = Base.rte/Effects/Glows/ExplosionHuge.bmp
 	FireSound = Sound
 		AddSample = ContentFile
-			FilePath = Imperatus.rte/Devices/Weapons/Marauder/Sounds/MarauderFire.wav
+			FilePath = Imperatus.rte/Devices/Weapons/Marauder/Sounds/Fire1.wav
+		AddSample = ContentFile
+			FilePath = Imperatus.rte/Devices/Weapons/Marauder/Sounds/Fire2.wav
+		AddSample = ContentFile
+			FilePath = Imperatus.rte/Devices/Weapons/Marauder/Sounds/Fire3.wav
 	EmptySound = Sound
 		AddSample = ContentFile
 			FilePath = Base.rte/Sounds/Devices/EmptyClick1.wav
 	ReloadStartSound = Sound
 		AddSample = ContentFile
-			FilePath = Base.rte/Sounds/Devices/ReloadStart.wav
+			FilePath = Imperatus.rte/Devices/Weapons/Marauder/Sounds/ReloadStart.wav
```

Comment:
I'm thinking this should have a chonky shell-by-shell reload.

Reply:
sprite looked like it had a weird unused magazine to me, i'll wait for further opinions

Reply:
I think the concept of firing hot chain is a bit unorthodox to consolidate a shell-by-shell reload.